mentoring/programs/the1-on-1hour.md

```diff
@@ -23,7 +23,7 @@ We will attempt to match you with the closest mentor of your area of interested
 
 ## Mentors
 
-If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](mentor-guide.md).
+If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](../processes/mentor-guide.md).
```

Comment:
```suggestion
If you are interested in signing up to be a mentor for this, please see the [mentor guidelines](/mentoring/processes/mentor-guide.md).
```
Using the full path from the root is preferred. It ensures if the doc is ever moved again the link won't have to be fixed.

Reply:
yes, make sense.
Update the PR.